src/midi/midi.cpp

```diff
@@ -203,6 +203,30 @@ void MIDI_RawOutByte(uint8_t data)
 	}
 }
 
+void MidiHandler::HaltSequence()
+{
+	uint8_t message[3] = {}; // see MIDI_evt_len for length lookup-table
+	constexpr uint8_t all_notes_off = 0x7b;
+	constexpr uint8_t all_controllers_off = 0x79;
+
+	// from the first to last channel
+	for (uint8_t channel = 0xb0; channel <= 0xbf; ++channel) {
+		message[0] = channel;
+
+		message[1] = all_notes_off;
+		PlayMsg(message);
+
+		message[1] = all_controllers_off;
+		PlayMsg(message);
+	}
```

Comment:
Won't this send random garbage in the third byte? Probably doesn't matter, but I'm slightly uncomfortable with it 😄 Better to explicitly set to zero, as suggested [here](https://users.cs.cf.ac.uk/dave/Multimedia/node158.html)?

Reply:
Line 208 initializes the array:
```
uint8_t message[3] = {};
```

Which assigns zero to all elements (so `[message[2]` is zero).

Reply:
Right, good to know. My C++ is quite rusty, indeed!

Reply:
I'd rather any of us be be rusty on details but sharp on intent (avoiding garbage/uninitialized values), and appreciate a second set of eyes and words of caution. Thanks, @johnnovak 👍